stock_orderpoint_automatic_creation/models/product.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Daniel Campos <danielcampos@avanzosc.es> - Avanzosc S.L.
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.model
+    def create(self, vals):
+        result = super(ProductProduct, self).create(vals)
+        orderpoint_obj = self.env['stock.warehouse.orderpoint']
+        if result.type != 'service':
+            orderpoint_obj.create(
+                {'name': result.name,
+                 'product_id': result.id,
+                 'product_max_qty':
+                 self.env.user.company_id.orderpoint_product_max_qty,
+                 'product_min_qty':
+                 self.env.user.company_id.orderpoint_product_min_qty
```

Comment:
Finally added default warehouse to company configuration so only one warehouse by company could be configured for automatic orderpoint creation, not sure if I should explain it in Known issues / Roadmap.